react/FileSelector/Readme.md

```diff
@@ -0,0 +1,48 @@
+This component is used to display a customizable file selector.
```

Comment:
We should clarify that we wrap an input type=file so we can pass all the related attributes like `accept` `mutliple` ... ?